### ISSUE TITLE
Sound is choppy on Game Boy Advance

### ISSUE THREAD

**Issue by askondro** [higan-emu/higan#168]
When playing Legend of Zelda - The link to the past & Four swords, the sound is always choppy. It does not matter which sound device I use in the settings.

**Comment by Screwtapello** [comment]
Is the emulation running at a solid 60 fps, according to the frame-rate counter in the bottom-right of the higan window?

**Comment by askondro** [comment]
It's about 57 - 59 FPS

**Comment by askondro** [comment]
drops to 54 fps sometimes.

**Comment by askondro** [comment]
Averager seems to be 55 FPS.

**Comment by Screwtapello** [comment]
Yep, if you're not getting steady 60fps, that would cause choppy sound.

higan's GBA emulation is known to be not as efficient as it could be. You might have better luck running the game in [mGBA](https://mgba.io/).

**Comment by askondro** [comment]
mGBA Works  great on linux. Not so much on windows.
